cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -0,0 +1,314 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import math
+import pytest
+import re
+
+from datetime import date
+
+from cfme import test_requirements
+from cfme.base.credential import Credential
+from cfme.common.vm import VM
+from cfme.common.provider import BaseProvider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.intelligence.reports.reports import CustomReport
+from cfme.utils.log import logger
+from cfme.utils.wait import wait_for
+
+
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.ignore_stream('5.8'),
+    pytest.mark.provider([VMwareProvider, RHEVMProvider],
+                         scope='module',
+                         required_fields=[(['cap_and_util', 'test_chargeback'], True)]),
+    test_requirements.chargeback,
+]
+
+
+@pytest.yield_fixture(scope="module")
+def clean_setup_provider(request, has_no_providers_modscope, setup_provider_modscope,
+        provider):
+    yield
+    BaseProvider.clear_providers()
+
+
+def new_credential():
+    return Credential(principal='uid' + '{}'.format(fauxfactory.gen_alphanumeric()),
+        secret='secret')
+
+
+@pytest.yield_fixture(scope="module")
+def vm_ownership(enable_candu, clean_setup_provider, provider, appliance):
+    # In these tests, chargeback reports are filtered on VM owner.So,VMs have to be
+    # assigned ownership.
+
+    vm_name = provider.data['cap_and_util']['chargeback_vm']
+
+    if not provider.mgmt.does_vm_exist(vm_name):
+        pytest.skip("Skipping test, cu-24x7 VM does not exist")
+    if not provider.mgmt.is_vm_running(vm_name):
```

Comment:
**Required**

`RHEVMSystem.start_vm()` and `VMWareSystem.start_vm()` already check if the VM is running, start if necessary, and call wait_vm_running. No need to wrap that logic again here, you can just call `start_vm()`

Reply:
Done